### PR TITLE
fix: expect ast in frontmatter

### PR DESCRIPTION
**Pull request by marbemac**
Entire parsed result is not needed.

Also, update sl-scripts to latest.

### CODE REVIEW

File: src/frontmatter/frontmatter.ts [hunk 1]
```diff
@@ -1,26 +1,20 @@
 import * as yaml from 'js-yaml';
-import _get = require('lodash/get');
-import _pullAt = require('lodash/pullAt');
-import _set = require('lodash/set');
-import _toPath = require('lodash/toPath');
-import _unset = require('lodash/unset');
+import { get, pullAt, set, toPath, unset } from 'lodash';
```
Comment (P0lip) [inline src/frontmatter/frontmatter.ts:2]: Will this be tree-shaken?

**Review by P0lip — APPROVED** [review]
LGTM, but see my comment re Lodash.
I think it might beneficial to keep scoped imports.

File: src/frontmatter/frontmatter.ts (hunk 1 above)
Comment (marbemac) [reply src/frontmatter/frontmatter.ts:2]: I've tested it w babel, which we are using in dojo app and I'm introducing to studio build process, and yes it is tree shaken :). However, babel does not support the weird `import x = require(y);` type syntax, so I've been slowly removing that from our various repos.

Comment (P0lip) [reply src/frontmatter/frontmatter.ts:2]: > I've tested it w babel, which we are using in dojo app and I'm introducing to studio build process, and yes it is tree shaken :)

Sounds good, let's merge it in!

> import x = require(y);

Yeah, it's TS-specific.

**Pull request merged**